### PR TITLE
Delete keynode

### PR DESCRIPTION
**Pull request by FalseR20**
# Delete keynode from kb and ScKeynodes

## How to use

```python
ScKeynodes.delete("idtf_to_delete")
```

### CODE REVIEW

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: tests/test_keynodes.py [hunk 1]
```diff
@@ -27,5 +27,11 @@ def test_resolve_keynode(self):
         self.assertTrue(delete_elements(addr))
         self.assertTrue(addr.is_valid())
 
+    def test_delete_keynode(self):
+        idtf = "idtf_to_delete_keynode"
+        ScKeynodes.resolve(idtf, sc_types.NODE_CONST)
+        del ScKeynodes[idtf]
+        self.assertFalse(ScKeynodes.get(idtf).is_valid())
+
```
[the remark this answers is not in the document]
Comment (FalseR20) [reply tests/test_keynodes.py]: Maybe we need to add refresh method that updates all stored keynodes addresses?

[the remark this answers is not in the document]
Comment (AlexandrZagorskiy) [reply tests/test_keynodes.py]: Okay, got it. So, we can implement **.pop()** method, if you want to propose logic as dict. I don't like the idea of providing deleting function as non-method. We can discuss this case in PM I think

[the remark this answers is not in the document]
Comment (AlexandrZagorskiy) [reply tests/test_keynodes.py]: Add refresh method looks like good idea, but naming is not clear, I prefer **sync()** method name for syncing with KB state. It's hard to understand when should I use it. Maybe it should be incapsulated and used automatically?

[the remark this answers is not in the document]
Comment (FalseR20) [reply tests/test_keynodes.py]: I decided to use `delete` method name, it looks prettier and better.
`sync` method will be added in other PR

**Review by AlexandrZagorskiy — APPROVED** [review]
Update changelog

**Pull request merged**